Update philips_js.markdown (#37891)
home-assistant/home-assistant.github.io: source/_integrations/philips_js.markdown

The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API.

## Prerequisites

Before adding the integration, it is worth checking if this integration works for you. As there are differences between TVs, you may need to try different addresses:

> - Some TVs use `https://`, other TVs use `http://`.
> - Some TVs primarily use port 1926, other TVs use port 1925 only.

Reviewer: ```suggestion
- Some TVs use `https://` and port 1926, other TVs use `http://` and port 1925.
```

### Check if this integration can be used

1. Check if your TV responds to one of the following combinations:
    - `http://IP_ADDRESS_OF_TV:1925/system`
    - `https://IP_ADDRESS_OF_TV:1926/system`
2. If you get a response, then this integration can be used. 
   - In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).

> 3. For older TVs follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). 

Reviewer: ```suggestion
3. For older TVs, follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). 
```

## Prerequisites

Before adding the integration, it is worth checking if this integration works for you. As there are differences between TVs, you may need to try different addresses:

- Some TVs use `https://`, other TVs use `http://`.
- Some TVs primarily use port 1926, other TVs use port 1925 only.

### Check if this integration can be used

1. Check if your TV responds to one of the following combinations:
    - `http://IP_ADDRESS_OF_TV:1925/system`
    - `https://IP_ADDRESS_OF_TV:1925/system`
    - `http://IP_ADDRESS_OF_TV:1926/system`
    - `https://IP_ADDRESS_OF_TV:1926/system`
2. If you get a response, then this integration can be used. 
   - In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
3. For older TVs follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). 
   - Note that not all listed, jointSPACE-enabled devices will have JSON-interface running on port 1925. 
   - This is true at least for some models before year 2011.

>     - Also, note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.

Reviewer: ```suggestion
   - Also, note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.
```

The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API.

> **Differences between Android TVs and non-Android TVs**
> 1) Android TVs use `https://`, non-Android TV use `http://`
> 2) Android TVs primarily use port 1926, non-Android TV use port 1925 only

Reviewer: ```suggestion

**Differences between Android TVs and non-Android TVs**: 

1. Android TVs use `https://`, non-Android TV use `http://`.
2. Android TVs primarily use port 1926, non-Android TVs use port 1925 only.
```

> For non-Android TV check if your TV responds to `http://IP_ADDRESS_OF_TV:1925/system`. 

Reviewer: ```suggestion
If you have a non-Android TV, check if your TV responds to `http://IP_ADDRESS_OF_TV:1925/system`. 
```

> For Android TV check if your TV responds to `https://IP_ADDRESS_OF_TV:1926/system`.

Reviewer: ```suggestion
If you have an Android TV, check if your TV responds to `https://IP_ADDRESS_OF_TV:1926/system`.
```

> If you get a response then this integration can be used. In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).

Reviewer: ```suggestion
If you get a response, then this integration can be used. In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
```

The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API.

**Differences between Android TVs and non-Android TVs**: 

1. Android TVs use `https://`, non-Android TV use `http://`.

> 2. Android TVs primarily use port 1926, non-Android TVs use port 1925 only.

Reviewer 2: This is incorrect as far as i know..Titan TVs use 1926 too i think

Author: Is it Titan TV a Philips model?
I updated the Philips TV documentation based on my experience http://xx.xx.xx.xx:1925/system was not working for my android Philips TV. I found a solution and shared it in the docs.

Reviewer 2: It is the OS on the tv. Android, Saphi, titan and some unknown legacy OS.

Some use https (1926). As far as i know they all respons on 1925 for system info. But some can only be controlled on 1926 after auth.

So keep it generic.

Author: Thanks, I made it generic.

The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API.

> 
> ## Differences between TVs 
> 
> 1. Some TVs use `https://`, other TVs use `http://`.
> 2. Some TVs primarily use port 1926, other TVs use port 1925 only.
> 
> ## Check if this integration can be used
> Check if your TV responds to one of the following combinations:
> 
> - `http://IP_ADDRESS_OF_TV:1925/system`
> - `https://IP_ADDRESS_OF_TV:1925/system`
> - `http://IP_ADDRESS_OF_TV:1926/system`
> - `https://IP_ADDRESS_OF_TV:1926/system`
> 
> If you get a response, then this integration can be used. In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).

Reviewer: ```suggestion

## Prerequisites

Before adding the integration, it is worth checking if this integration works for you. As there are differences between TVs, you may need to try different addresses:

- Some TVs use `https://`, other TVs use `http://`.
- Some TVs primarily use port 1926, other TVs use port 1925 only.

### Check if this integration can be used

1. Check if your TV responds to one of the following combinations:
    - `http://IP_ADDRESS_OF_TV:1925/system`
    - `https://IP_ADDRESS_OF_TV:1925/system`
    - `http://IP_ADDRESS_OF_TV:1926/system`
    - `https://IP_ADDRESS_OF_TV:1926/system`
2. If you get a response, then this integration can be used. 
   - In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
```

The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API.

## Prerequisites

Before adding the integration, it is worth checking if this integration works for you. As there are differences between TVs, you may need to try different addresses:

- Some TVs use `https://`, other TVs use `http://`.
- Some TVs primarily use port 1926, other TVs use port 1925 only.

### Check if this integration can be used

1. Check if your TV responds to one of the following combinations:
    - `http://IP_ADDRESS_OF_TV:1925/system`

>     - `https://IP_ADDRESS_OF_TV:1925/system`

Reviewer 2: ```suggestion
```

>     - `http://IP_ADDRESS_OF_TV:1926/system`

Reviewer 2: ```suggestion
```

    - `https://IP_ADDRESS_OF_TV:1926/system`
2. If you get a response, then this integration can be used. 
   - In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).

> For older TVs follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). Note that not all listed, jointSPACE-enabled devices will have JSON-interface running on port 1925. This is true at least for some models before year 2011.

Reviewer: ```suggestion
3. For older TVs follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). 
   - Note that not all listed, jointSPACE-enabled devices will have JSON-interface running on port 1925. 
   - This is true at least for some models before year 2011.
```

> 
> Also, note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.

Reviewer: ```suggestion
    - Also, note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.
```
